sdk/loadtestservice/load-testing-rest/README.md

```diff
@@ -120,46 +126,124 @@ await client.path("/loadtests/{testId}", TEST_ID).patch({
 ### Uploading .jmx file to a Test
 
 ```javascript
-import { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { AzureLoadTestingClient, beginUploadTestFile } from "@azure-rest/load-testing";
```

Comment:
Could you update the samples here to match the latest public API updates?

Reply:
Updated Readme Samples.